lib/eChronicler_web/templates/hello/index.html.heex

```diff
@@ -0,0 +1,13 @@
+<div class="phx-hero">
+  <h2>Enter your journal entry.</h2>
+
+    <form>
+      <label for="title">Title:</label><br>
+  <input type="text" id="title" name="title"><br>
+  <label for="author">Author:</label><br>
+  <input type="text" id="author" name="author">
+    <label for="Entry">Entry:</label><br>
+  <textarea type="text" id="entry" name="entry"></textarea>
+  <input type="submit" value="Submit">
+    </form>
```

Comment:
Indentation off here? And is this for creating a new page?  Maybe it's something you tried but had to be put on the backburner?  Try to limit the scope of the PR to only the list-all/index functionality.  You can delete or perhaps  "stash" unused things, for example, by deleting this file and the "show" method in the controller above.  

If you are deleting things you will need later for another story: One possible option is to delete things, and save the deletions (only the deletions) in a commit.  Later, when you start a new branch, run `git revert <commit sha>` to REVERSE-APPLY the deletions in this commit.  There may be a better way to do it, but that popped into my brain.

https://git-scm.com/docs/git-revert